src/redux/services/validation.ts

```diff
@@ -54,6 +54,35 @@ export function validateResource(resource: K8sResource) {
     return;
   }
 
+  const errors = [];
+
+  // parse for YAML errors that were allowed by non-strict parsing
+  const doc = parseDocument(resource.text);
+  if (doc.errors.length > 0) {
+    const lines = resource.text.split('\n');
+    if (lines[0] === '---') {
+      lines.shift();
+    }
+
+    errors.push(
+      ...doc.errors.map(err => {
+        const line = err.linePos ? lines[err.linePos[0].line].trim() : '';
```

Comment:
is it possible for `err.linePos[0].line` to be undefined? maybe we should do a sanity check